projects/fhi-angular-components/src/lib/fhi-time/fhi-time-constants.ts

```diff
@@ -1,7 +1,12 @@
 export class FhiTimeConstants {
-
   // TODO: i18n
-  static weekpickerPlaceholder = 'åååå-u[u]';
+  static weekpickerPlaceholder = 'åååå-uu';
   static weekpickerLabel = 'Velg uke';
   static weekpickerDelimiter = '-';
+
+  static weekRangeLabelFrom = 'Fra uke';
+  static weekRangeLabelTo = 'Til uke';
+
+  static maxWeek = { year: new Date().getFullYear() + 1, week: 52 };
```

Comment:
Why year + one?

Reply:
In the odd case if currently in week 53, this is the simple solution that could be a candidate for improvement and set maxWeek to 53 current year.